frontend/.eslintignore

```diff
@@ -5,3 +5,4 @@ public/dist
 public/lib
 **/node_modules
 Godeps
+integration-tests-cypress
```

Comment:
We should lint our tests.

Reply:
If I remove that line I get:
```
/home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: integration-tests-cypress/cypress/tests/monitoring_spec.ts.
The file must be included in at least one of the projects provided

/home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tsconfig.json
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: integration-tests-cypress/cypress/tsconfig.json.
The file must be included in at least one of the projects provided
```
If I remove that line and stop excluding `integration-tests-cypress` from `frontend/tsconfig.json` then it should fix the eslint problem, but then when I try `build-frontend.sh` I get:
```
ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/support/index.d.ts
(10,47): Type 'Chainable' is not generic.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(1,54): File '/home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/support/index.d.ts' is not a module.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(5,3): Cannot find name 'cy'.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(6,3): Cannot find name 'cy'.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(8,5): Cannot find name 'cy'.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(17,3): Cannot find name 'before'.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(18,5): Cannot find name 'cy'.

ERROR in /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/integration-tests-cypress/cypress/tests/monitoring_spec.ts
(20,3): Cannot find name 'after'.

```

Reply:
We can't disable linting. Make sure you're including the TypeScript declarations for Cypress:

https://github.com/cypress-io/cypress/tree/develop/cli/types

Reply:
Fixed, see [comment](https://github.com/openshift/console/pull/4831#issuecomment-611705992)